### PR TITLE
Adds Ork subtype of hulk mutation with funny speech modifiers

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Adds a subtype of hulk mutation created by mixing it with clumsy. Makes the owner's skin olive green and forces a funny speech impediment on them. Essentially revival of #69473 with some more lines
изображение

## Why It's Good For The Game

WARBOYZ NEEDA MOAR DAKKA

funny flavor subtype for hulks which makes them sound even more brain damaged

## Changelog
:cl:
add: A new ork mutation which can be acquired by mixing hulk and clumsy. 
/:cl:

### CODE REVIEW

File: strings/ork_replacement.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		"engineering": "tinkering",
+		"technician": "mekboy",
+		"botanist": "green feeder",
+		"botany": "green feed",
+		"felinid": "talking kitty",
+		"robotics": "kanna maka",
+		"roboticist": "kanna boy"
+		"cat": "kitty",
+		"moth": "bug",
+		"mothperson": "bugperson",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:33]:
```suggestion
		"mothperson": "bugman",
```
person has too many syllables, "bugboy" or just "bug" is also acceptable

File: strings/ork_replacement.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+		"pistol": "short boomstick",
+		"shotgun": "boomstick",
+		"launcher": "bomb throwa",
+		"sword": "harder choppa",
+		"knife": "choppa",
+		"spear": "long choppa",
+		"disk": "disky",
+		"nuke disk": "disky",
+		"captain": "warboss",
+		"security" : "warboyz",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:18]:
```suggestion
		"security": "warboyz",
```

File: strings/ork_replacement.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+		"cap ": "boss ",
+		" cap": " boss",
+		"science": "smarty thinking",
+		"scientist" :"brain boy",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:22]:
```suggestion
		"scientist": "brain boy",
```

File: strings/ork_replacement.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+		"technician": "mekboy",
+		"botanist": "green feeder",
+		"botany": "green feed",
+		"felinid": "talking kitty",
+		"robotics": "kanna maka",
+		"roboticist": "kanna boy"
+		"cat": "kitty",
+		"moth": "bug",
+		"mothperson": "bugperson",
+		"plasmaman": "fire skeleton",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:34]:
```suggestion
		"plasmaman": "fire skellyton",
```

File: strings/ork_replacement.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+		"plasmamen": "fire skeletons",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:35]:
```suggestion
		"plasmamen": "fire skellytons",
```

Comment (Jacquerel) [reply strings/ork_replacement.json:35]: (and of course the same for "skeleton")

File: strings/ork_replacement.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+		"fight": "crump",
+		"revolution": "dust up",
+		"nuke": "bomb",
+		"ai ": "talky machine ",
+		"artificial intelligence": "talky machine",
+		"shuttle": "winged trukk",
+		"miner": "rockcrusher",
+		"mining": "rock crushing",
+		"maintenance": "tunnel",
+		"maint":"tunnel",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:54]:
```suggestion
		"maint": "tunnel",
```

File: strings/ork_replacement.json [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+		"fly": "buzz",
+		"flyperson": "buzzer",
+		"run": "charge",
+		"fight": "crump",
+		"revolution": "dust up",
+		"nuke": "bomb",
+		"ai ": "talky machine ",
+		"artificial intelligence": "talky machine",
+		"shuttle": "winged trukk",
+		"miner": "rockcrusher",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:51]:
```suggestion
		"miner": "rockkrusha",
```

File: strings/ork_replacement.json [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+		"mining": "rock crushing",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:52]:
```suggestion
		"mining": "rock krushing",
```

File: strings/ork_replacement.json [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+		"disk": "disky",
+		"nuke disk": "disky",
+		"captain": "warboss",
+		"security" : "warboyz",
+		"cap ": "boss ",
+		" cap": " boss",
+		"science": "smarty thinking",
+		"scientist" :"brain boy",
+		"engineer": "mekkanik",
+		"engineering": "tinkering",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:24]:
```suggestion
		"engineering": "tinkerin'",
```

File: strings/ork_replacement.json [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+		"technician": "mekboy",
+		"botanist": "green feeder",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:26]:
```suggestion
		"botanist": "green feeda",
```

File: strings/ork_replacement.json [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+		"felinid": "talking kitty",
+		"robotics": "kanna maka",
+		"roboticist": "kanna boy"
+		"cat": "kitty",
+		"moth": "bug",
+		"mothperson": "bugperson",
+		"plasmaman": "fire skeleton",
+		"plasmamen": "fire skeletons",
+		"lizard": "tailswinger",
+		"lizardperson": "tailswinger",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:37]:
```suggestion
		"lizard": "tailswinga",
		"lizardperson": "tailswinga",
```

File: strings/ork_replacement.json [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+		"car": "trukk",
+		"power": "elektricity",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:39]:
```suggestion
		"power": "lektricity",
```

File: strings/ork_replacement.json [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+		"cable": "zap carrier",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:40]:
```suggestion
		"cable": "zap rope",
```

File: strings/ork_replacement.json [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+		"human": "humie",
+		"fly": "buzz",
+		"flyperson": "buzzer",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:43]:
```suggestion
		"flyperson": "buzza",
```

File: strings/ork_replacement.json [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+		"run": "charge",
+		"fight": "crump",
+		"revolution": "dust up",
+		"nuke": "bomb",
+		"ai ": "talky machine ",
+		"artificial intelligence": "talky machine",
```
Comment (Jacquerel) [inline strings/ork_replacement.json:49]:
```suggestion
		"artificial intelligence": "talky box",
```

Comment (Jacquerel) [inline strings/ork_replacement.json:49]:
```suggestion
		"ai ": "talky box ",
		"artificial intelligence": "talky box",
```

**Review by Jacquerel — COMMENTED** [review]
Many optional suggestions, some weird spacing going on also.
To be honest I am a little ambivalent about the idea of hulks talking like orks, it's funny but it's also a very recognisable reference. I probably won't merge this but I don't feel strongly enough to block it either.

File: strings/ork_replacement.json (hunk 1 above)
Comment (Jacquerel) [reply strings/ork_replacement.json:33]: or of course "bugga" but that might give people the wrong idea

File: strings/ork_replacement.json (hunk 11 above)
Comment (Jacquerel) [reply strings/ork_replacement.json:37]: maybe the ones for species would look better with an h on the end, idk

**Comment by Jacquerel** [comment]
Won't come up often but you're missing `teeth = teef`

File: strings/ork_replacement.json (hunk 11 above)
Comment (EricZilla) [reply strings/ork_replacement.json:37]: I like the H on there

**Comment by Jacquerel** [comment]
Other suggestions I found online:
```
proper = proppa
something = sumfing
nothing = nuffin
I am = I iz
I'm = I'ze
We're = We'ze
You're = You'ze
This = Dis
That = Dat
Them = Dem
Those = Dose
Your = Ya
My = Me
Says = Sez
With = Wiv
What = Wot
Who = Oo
Who's/Whose = Oo'z
Strong / Tough / Hard = 'ard
Robust = well 'ard
What's that = wuzzat
```

**Comment by Rex9001** [comment]
Maybe Heretic and Wizard -> Weirdboy

File: strings/ork_replacement.json [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+		"war ": "WAAAGH! ",
+		"hos ": "nob ",
+		"the ": "da ",
+		".": "!",
+		"one ": "wun ",
+		"fast": "fastah",
+		"faster": "fastah",
+		"exosuit": "killa kan",
+		"mech": "killa kan",
+		"mecha": "killa kan",
```
Comment (Iamgoofball) [inline strings/ork_replacement.json:66]: why is this not mek

**Comment by SomeoneYouProbablyKnow** [comment]
I gotta say it, pretty mid 40k reference, especially considering hulks can't even use guns (it would be really funny if stun immunity wasn't as stupid and you could give em stormtrooper aim instead). Very well made tho, good job

**Comment by kawaiinick** [comment]
they are hulks not orcs

**Comment by carlarctg** [comment]
make this a combination mut of hulk x stoner that gives orkspeeech and hylk

**Comment by MrMelbert** [comment]
Hulks are not orks, this is just a reference that completely misses the mark

**Comment by carlarctg** [comment]
which is why it'd be fine as a mutation of hulk but not as hulk proper!

this would also prevent the joke from getting stale

**Comment by Iamgoofball** [comment]
>Hulks are not orks, this is just a reference that completely misses the mark

oranges made this decision, take it up with him

**Comment by SmArtKar** [comment]
I'm 50/50 on making it a subtype of hulk, so up to maintainers I guess

**Comment by EricZilla** [comment]
It would be funny if you had a slim chance of getting orkspeak instead of yelling when gaining the mutation

**Comment by Ghommie** [comment]
> I'm 50/50 on making it a subtype of hulk, so up to maintainers I guess

i'm cool with it sticking to hulk without a subtype.

**Comment by Iamgoofball** [comment]
okay, but that's going against a headcoder proposed design

**Comment by Ghommie** [comment]
That's also true that it kinda misses the mark.

**Comment by SmArtKar** [comment]
Converted this to be a separate subtype mutation created by mixing hulk and clumsy per melbert's suggestion

**Comment by carlarctg** [comment]
MY suggestion not melberys

**Comment by optimumtact** [comment]
This shouldnt' have been converted to a subtype...

**Comment by SmArtKar** [comment]
krill issue, melbert is to be blamed

**Pull request merged**